packages/eos-components/src/index.scss

```diff
@@ -95,6 +95,22 @@
   white-space: pre-wrap;
 }
 
-.card.disabled {
-  opacity: 0.7;
+.welcome-card {
```

Comment:
After a second thought, I think all this CSS should go to a WelcomeCard component. See all other styles in this `index.scss` files, they are not for a custom component in particular: they affect all buttons, all breadcrumbs, etc. If this were CSS for all cards then it should go here. But it's for a particular kind of cards.

Reply:
Yes, that's a better solution, we can implement the `WelcomeCard` to share the same css in a component instead of a class.

Reply:
I'll create the `WelcomeCard` component to use here.